app/src/androidTest/java/net/skyscanner/backpack/compose/text/BpkTextTest.kt

```diff
@@ -29,6 +32,21 @@ class BpkTextTest : BpkSnapshotTest() {
 
   @Test
   fun default() = composed {
-    Text(text = "Sample")
+    DefaultText()
```

Comment:
something i thought of when setting up tests here: maybe we should have previews for what we test as that might be helpful when making changes to quickly verify different properties